lib/components/STable.vue

```diff
@@ -225,62 +224,81 @@ function getCell(key: string, index: number) {
         </div>
 
         <div
-          v-if="!loading && records && records.length"
+          v-if="!unref(options.loading) && unref(options.records)?.length"
           class="container body"
           ref="body"
           @mouseenter="lockBody(true)"
           @mouseleave="lockBody(false)"
           @scroll="syncBodyScroll"
         >
-          <div class="block">
-            <div
-              v-for="(record, rIndex) in recordsWithSummary"
-              :key="rIndex"
-              class="row"
-              :class="isSummary(rIndex) && 'summary'"
-            >
-              <STableItem
-                v-for="key in orders"
-                :key="key"
-                :name="key"
-                :class-name="columns[key].className"
-                :width="colWidths[key]"
+          <DynamicScroller
+            :items="recordsWithSummary"
+            :min-item-size="40"
```

Comment:
might need to change this value too 👀

Reply:
Ah. What does this do? It sets min height...?

Reply:
Minimum height of a row.

Reply:
Gotcha. I'll set this to match the height of the body when spinner is shown 👍 (when `loading: true`.

Reply:
Ah yeah so that's kinda inevitable because sometimes we only have 1 record 😅 (depending on applied filters and such).

And we always get layout shift when paginating due to showing spinner 👀